### ISSUE TITLE
Option to export only members in subclasses that differ from their parent's version

### ISSUE THREAD

**Issue by JPLeBreton** [mitmproxy/pdoc#106]
I have class A which has a lot of members, and class B which extends A and overrides/adds a single function or variable. I'd prefer the documentation for B to only show documentation for that one member, and not reproduce all the other members from A which are identical. In cases where class A has several trivial subclasses, documentation files contain massive amounts of completely redundant information. Is it possible to detect these cases?

**Comment by BurntSushi** [comment]
@JPLeBreton You might be able to do this by modifying the template. See here for example: https://github.com/BurntSushi/pdoc/blob/master/pdoc/templates/html.mako#L177-L181

**Comment by JPLeBreton** [comment]
Just wanted to check that what I'm seeing is the intended behavior:

http://vectorpoem.com/playscii/generated/game_util_objects.html

Here are some of the generated docs for a project of mine. On that page are several smallish utility classes that extend from a base class (GameObject) with a ton of methods and functionality. 99% of that page is duplicated doc strings for the non-overridden methods of the parent class. Is this actually the desired default behavior, for any project? My documentation page there is significantly less useful because of all the redundant text. A link to the doc entry for the parent class at the top of each subclass would suffice.

**Comment by mhils** [comment]
Hi @JPLeBreton,

I believe the documentation you linked was built with pdoc3, not pdoc (see [pdoc3 vs. pdoc3](https://github.com/mitmproxy/pdoc#pdoc-vs-pdoc3)).

Regarding your question, if you inherit from a base class and do not overwrite the method, pdoc should only list the method name (not the docstring) in the "Inherited" section. If you do overwrite the method without specifying a new docstring, the docstring is inherited. 

If you don't like that, you can either insert empty docstrings in your code (ugly) or modify the template to only display for docstrings for some members. :)

**Comment by JPLeBreton** [comment]
> I believe the documentation you linked was built with pdoc3, not pdoc (see [pdoc3 vs. pdoc3](https://github.com/mitmproxy/pdoc#pdoc-vs-pdoc3)).

!!!! Oh, my. Well, that would explain a lot! I integrated pdoc into my project back in mid 2016 or so, when it appears the module worked totally differently. I've just upgraded to your new release, and changed my program to use the module correctly. The newly generated docs look great and its way of presenting inherited stuff is ideal. Thanks for cluing me in to the change!
